scripts/install_third_party_libs_test.py

```diff
@@ -348,15 +348,16 @@ def mock_install_skulpt(unused_parsed_args):
             check_function_calls['install_skulpt_is_called'] = True
         def mock_check_call(unused_cmd_tokens):
             pass
-        def mock_main_for_install_third_party(unused_args):
+        # pylint: disable=unused-argument
+        def mock_main_for_install_third_party(args):
             check_function_calls['install_third_party_main_is_called'] = True
-        def mock_main_for_setup(unused_args):
+        def mock_main_for_setup(args):
```

Comment:
Those have to be `args`, as those functions are explicitly invoked with `args` parameter. 
https://github.com/oppia/oppia/blob/eed3cf4cf7cc665e9618f343fc45113d61a44799/scripts/install_third_party_libs.py#L286